src/Features/CSharpTest/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -590,7 +590,7 @@ class Program
                 static void Main()
                 {
                     byte z = 0;
-                    Goo({|Rename:NewMethod|}(), y => (byte)0, z, z);
+                    Goo<byte, byte>({|Rename:NewMethod|}(), y => 0, z, z);
```

Comment:
50/50 on which is better.  because we now determine we can remove the cast, the generic stay required.